src/bluez/adapter.rs

```diff
@@ -22,6 +23,13 @@ impl Adapter {
     }
 }
 
+fn get_central_state(state: bool) -> CentralState {
```

Comment:
Nit: Call the parameter `powered` rather than `state`.

Reply:
done [bcf31a7](https://github.com/deviceplug/btleplug/pull/366/commits/bcf31a758619ffb71eeec2f0482a424e5ad9a40f)